app/models/host.rb

```diff
@@ -944,11 +944,11 @@ def self.ost_inspect(ost)
   end
 
   def rediscover(ipaddr, discover_types = [:esx])
-    require 'discovery/MiqDiscovery'
+    require 'manageiq-network_discovery'
```

Comment:
Does this work?  I don't see a `lib/manageiq-network_discovery.rb`in the gem.  Typically that file requires the rest.